gettsim/tests/test_elterngeld.py

```diff
@@ -58,14 +58,18 @@ def test_eltgeld(
     """
     year_data = input_data[input_data["jahr"] == year]
     df = year_data[INPUT_COLS].copy()
-    params, policy_functions = set_up_policy_environment(date=year,)
+    policy_params, policy_functions = set_up_policy_environment(date=year,)
     df["soli_st_tu"] = df["soli_st_m"].groupby(df["tu_id"]).transform("sum") * 12
     df["eink_st_tu"] = df["eink_st_m"].groupby(df["tu_id"]).transform("sum") * 12
 
-    columns = ["soli_st_tu", "sozialv_beitr_m"]
+    columns_overriding_functions = ["soli_st_tu", "sozialv_beitr_m", "eink_st_tu"]
 
     result = compute_taxes_and_transfers(
-        df, params=params, targets=column, columns_overriding_functions=columns
+        df,
```

Comment:
Here and elsewhere: I do not think there is a good reason to mix location-based and name-based passing of parameters. The latter is clearer, is it too much to ask for some block copy & pastes?

Reply:
I can do it. I would say whenever the variable name diverges from the argument name, it is more useful than when both are the same. And it is also pretty common to use common arguments as positional arguments. (See how pandas uses data and index ([docs](https://pandas.pydata.org/pandas-docs/stable/user_guide/indexing.html))).

Reply:
Yip, I just doubt that anybody will ever use GETTSIM at a similar frequency as Pandas :wink:, plus it is less obvious what the order of data / params / functions / targets should be.

In any case, thanks!